pyugrid/read_netcdf.py

```diff
@@ -238,8 +240,6 @@ def load_grid_from_nc_dataset(nc, grid, mesh_name=None, load_data=True):
             attributes = {n: var.getncattr(n) for n in var.ncattrs()
                           if n not in ('location', 'coordinates', 'mesh')}
 
-            # Trick with the name: FIXME: Is this a good idea?
-            name = name.lstrip(mesh_name).lstrip('_')
```

Comment:
I did remove this. If found it a bit confusing when writing and reading files. As far as I could tell, the mesh name was not prepended when writing data set variables. So had a situation like this: mesh2_face_ids --> face_ids --> ids.